### ISSUE TITLE
Unable to add new maintainer to go-maintainers GitHub team

### ISSUE THREAD

**Issue by achew22** [bazelbuild/bazel#15644]
### Description of the bug:

Members of the `bazelbuild/go-maintainers` team are unable to add new maintainers.

### What's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

1. Go to https://github.com/orgs/bazelbuild/teams/go-maintainers/members as a member of the go-maintainers team
2. Click "Add a member"
3. Type in the name of the maintainer you would like to add
4. Receive a message like this

<img width="805" alt="image" src="https://user-images.githubusercontent.com/37206/172758440-0f9449fb-b7ac-447d-8971-65ba2fca8e41.png">

I chose to put `@ghost` as the user for the screenshot for the party's privacy. If this requires someone from the Bazel team to  click a button, I'm happy to out of band share their username.

**Comment by achew22** [comment]
@sventiffe and @aiuto (sorry to ping you, but I know you helped out with this switchover), is there any update on this?

**Comment by Wyverald** [comment]
According to my read of https://docs.github.com/en/organizations/organizing-members-into-teams/adding-organization-members-to-a-team, the person needs to be a member of the bazelbuild organization first -- otherwise, you need admin access to add them to the org, and then to invite them to the team. I'm happy to assist with this if you email me the person's GitHub username.

**Comment by aiuto** [comment]
TL;DR;  +1 to what Yun said.

The idea was that they could self-maintain.
https://github.com/orgs/bazelbuild/teams/go-maintainers/members
shows Allan as a maintainer, so I don't understand why he can not add the
person.
Maybe it is just that the new person must be in bazelbuild first. We have
lots of non-googlers in ***@***.***

So, it should just be a question of adding them to bazelbuild first, then
Allan should be able to finish.
It is preferable if Allan tries to finish it that way, so we can prove that
it works as intended.

**Comment by achew22** [comment]
Where would one go to find this documentation?

**Comment by achew22** [comment]
Could we use this issue to track adding documentation on this point @aiuto or would you like me to file an additional issue?

**Comment by aiuto** [comment]
Did we ever add the new person?  I don't remember a mail thread about that.

**Comment by achew22** [comment]
The new person has been added, yes. Thank you @sventiffe! I would like to leave this open to track documenting the process
